standards/aws-tags.md

```diff
@@ -0,0 +1,35 @@
+# Using AWS Tags
+
+[Amazon's documentation](https://aws.amazon.com/answers/account-management/aws-tagging-strategies/) describes tags as:
+
+ >...a simple label consisting of a customer-defined key and an optional value that can make it easier to manage, search for, and filter resources. Although there are no inherent types of tags, they enable customers to categorize resources by purpose, owner, environment, or other criteria.
+
```

Comment:
There are actually some inherent types that get automagically assigned by the system.  I don’t remember if you have to turn that on or not, but once you do, it’s all there for all resources, I think?  They're just not tags that would be useful for budgeting...